src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,26 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() < 18) {
+            throw new RuntimeException("Age must be more than 17");
+        }
+        if (user.getPassword() == null || user.getPassword().length() < 6) {
+            throw new RuntimeException("Password must be more than 5 symbols");
```

Comment:
All magic number should be constants. Lets create **private static final int MINIMUM_PASSWORD_LENGTH = 6** field and use it instead of "6" in this case.

Reply:
Agree with Nikita, you do overmuch work in almost every test method